ogr/abstract.py

```diff
@@ -70,6 +114,21 @@ def __str__(self) -> str:
             f"created='{self.created}')"
         )
 
+    def can_close_issue(self, username: str) -> bool:
+        raise NotImplementedError()
+
+    def issue_comment(self, body: str) -> IssueComment:
+        raise NotImplementedError()
+
+    def close(self) -> None:
+        raise NotImplementedError()
+
+    def get_labels(self) -> List:
+        raise NotImplementedError()
+
+    def add_labels(self, labels: List[str]) -> None:
+        raise NotImplementedError()
+
```

Comment:
Do we want to separate *everything* issue related into this class, e.g. `get_issue_list`, `get_issue`, `create_issue`?

Reply:
I would leave that in the project class:

```python

my_new_issue = my_project.create_issue(...)

my_issue = my_project.get_issue(my_new_issue.id)
assert my_issue == my_new_issue
my_issue.comment(...)

assert my_project.get_issue_list()[-1] == my_new_issue
```